spec/features/user_signs_in_spec.rb

```diff
@@ -5,25 +5,25 @@
   I want to sign in
   So that I can regain access to my account
 } do
-  scenario 'specify valid credentials' do
+  scenario "specify valid credentials" do
     user = FactoryGirl.create(:user)
 
-    visit new_user_session_path
+    visit meetups_path
 
-    fill_in 'Email', with: user.email
-    fill_in 'Password', with: user.password
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
 
-    click_button 'Log in'
+    click_button "Sign in"
 
-    expect(page).to have_content('Signed in successfully')
-    expect(page).to have_content('Sign Out')
+    expect(page).to have_content("Signed in successfully")
+    expect(page).to have_content("Sign Out")
   end
 
-  scenario 'specify invalid credentials' do
+  scenario "specify invalid credentials" do
     visit new_user_session_path
 
-    click_button 'Log in'
-    expect(page).to have_content('Invalid email or password')
-    expect(page).to_not have_content('Sign Out')
+    click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.